js/noms/src/http-batch-store.js

```diff
@@ -52,7 +53,8 @@ const readBatchOptions = {
 export default class HttpBatchStore extends RemoteBatchStore {
   _rpc: RpcStrings;
 
-  constructor(urlparam: string, maxReads: number = 5, fetchOptions: FetchOptions = {}) {
+  constructor(
```

Comment:
Spec needs to set FetchOptions, and to pass that in it needs to pass through a maxReads value as well (just because of parameter ordering). This change is just exposing that `5` constant so that spec can pass in `5` itself.